src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+  hint: NoHint
+  explanation: CeFi favours only high net worth individuals because they are their potential customers who are bound to produce better returns than low worth individuals.
+  answerKeys:
+    - A
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1
+- uuid: 12d93c50-1c61-4f80-9ada-97f76b8eab1a
+  type: SingleChoice
+  content: What is the most popular use of DeFi? 
+  hint: NoHint
+  explanation: Decentralised Exchanges are the most popular use of DeFi
+  answerKeys:
+    - A
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: Decentralised Exchanges
+      key: A
+      order: 0
+    - content: Sending NFTs
+      key: B
+      order: 1
+    - content: Submitting job applications and rentals
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: cd152168-7ad7-4d34-b6f0-0797682ee35c
+  type: SingleChoice
+  content: In DeFi, staking rewards are generally paid in?
+  hint: NoHint
+  explanation: Staking rewards are generally paid out in the same cryptocurrency that you are staking. For instance, if you are staking ADA, you will get rewarded in ADA.
+  answerKeys:
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "High"
+  choices:
+    - content: USDC
+      key: A
+      order: 0
+    - content: USDT
+      key: B
+      order: 1
+    - content: The cryptocurrency you are staking
+      key: C
+      order: 2
+    - content: All of the above
+      key: D
+      order: 3
+- uuid: 85139b5a-282f-4f88-b4f8-e9f80544019b
+  type: SingleChoice
+  content: The main advantage of Decentralised Exchanges are?
+  hint: Properties of DEXs
+  explanation: Decentralized exchanges (DEXs) operate by connecting to a user’s external software wallet, which the user owns the private keys to. DEXs also boast greater privacy and anonymity due to the lack of KYC protocols, which makes them very popular with those who are concerned about privacy. DEXs also feature a large number of low cap altcoins, which can be diamonds in the rough or very risky investments.
+  answerKeys:
+    - D
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "High"
+  choices:
+    - content: Users own their private keys
+      key: A
+      order: 0
+    - content: Increased privacy due to lack of KYC
+      key: B
+      order: 1
+    - content: Large range of low cap alt coins
+      key: C
+      order: 2
+    - content: All of the above
+      key: D
+      order: 3
+- uuid: 360586a5-8a95-4566-a31e-caa447f78545
+  type: MultipleChoice
+  content: Which of the following are ways to earn returns on crypto assets?
+  hint: NoHint
+  explanation: Farming, Swapping and Staking are ways to earn interest and heavy returns on crypto assets.
+  answerKeys:
+    - D
```

Comment:
Should the correct choice be `All of the above` or `None of the above`?

Reply:
Done